docs/config.md

```diff
@@ -1281,7 +1281,7 @@ not being able to discover that you have the objects that you have. If you want
 to have this disabled and keep the network aware of what you have, you must
 manually announce your content periodically.
 
-Type: `array[peering]`
+Type: `string` (value which gets parsed into time.Duration) 
```

Comment:
```suggestion
Type: `duration`
```
https://github.com/ipfs/kubo/blob/d540d9a404fe16b805c4993c1a8d1b2e4f760e44/docs/config.md#duration

Reply:
Thanks @Jorropo! I didn't see that we had the types declared at the top of the file.